modules/ossm-configuring-jaeger-existing-v1x.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v1x/ossm-custom-resources.adoc
+
+[id="ossm-configuring-jaeger-existing-v1x_{context}"]
+= Connecting to an existing Jaeger instance
+
+In order for the SMCP to connect to an existing Jaeger instance, the following must be true:
+
+* The Jaeger instance is deployed in the same namespace as the control plane, for example, into the `istio-system` namespace.
+
+* To enable secure communication between services, you should enable the oauth-proxy, which secures communication to your Jaeger instance, and make sure the secret is mounted into your Jaeger instance so Kiali can communicate with it.
+
+* To use a custom or already existing Jaeger instance, set `spec.istio.tracing.enabled` to "false" to disable the deployment of a Jaeger instance.
+
+* Supply the correct jaeger-collector endpoint to Mixer by setting `spec.istio.global.tracer.zipkin.address` to the hostname and port of your jaeger-collector service. The hostname of the service is usually `<jaeger-instance-name>-collector.<namespace>.svc`.
```

Comment:
@JStickler I think we have to add `.svc.cluster.local` with the service name.
Example: `<jaeger-instance-name>-collector.<namespace>.svc.cluster.local`

@kevinearls @objectiser @jpkrohling Can you please confirm?

Reply:
If they're running in the same namespace I don't think you'd need to add cluster.local but can't say with 100% certainty.

Reply:
Both work in the default DNS lookup configuration. It makes a difference only (or "mostly") when dealing with TLS, which will do the host verification based on the hostname that was used in the connection string vs. the hostname from the certificate. If I remember correctly, the certs are issued for the `.svc.cluster.local` variant.

Reply:
@jpkrohling so is it OK as is, or do I need to add the example as @jkandasa suggested?

Reply:
To be on the safe side, follow @jkandasa's suggestion.